src/views/Signup.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import { commitMutation } from 'react-relay'
+import environment from '../config/relay'
+import { Redirect } from "react-router";
+import graphql from 'babel-plugin-relay/macro'
+
+import {Form, Button, Container} from "react-bootstrap";
+
+const mutation = graphql`
+    mutation SignUpMutation($input: UserCreateInput!){
```

Comment:
the relay compiler throws an error when trying to compile this. The case of `SignUp` doesn't match the filename.

Reply:
Actually I renamed it to SignUp.js, not sure why my commit didnt capture the rename

Reply:
Oh, that could be an OS issue with git. Linux and MacOS have different handling of case. MacOS is not case sensitive, Linux is. It's a cause of endless frustration. Do some Googling for how to resolve case issues with git, let me know if you have issues.